src/components/HURUmap/Panel/ProfileItems.js

```diff
@@ -51,42 +52,16 @@ const ProfileItems = memo(
             <CategoryHeader
               description={category?.description}
               icon={category.icon}
+              id={slugify(category.title)}
               title={category.title}
             />
             {category.children.map((child, subcategoryIndex) => (
               <Fragment key={child.title}>
                 <SubcategoryHeader
                   description={child?.description}
+                  id={slugify(child.title)}
```

Comment:
Just to confirm, it's not possible for **2** charts in _different_ categories to have the same name, right?

Reply:
@kilemensi yes for chart, that's possible. But we are using `id`s on chart. Or did you mean subcategory name for different category?  So yes, thats possible! Let me append the parent name